validator/sawtooth_validator/execution/scheduler_serial.py

```diff
@@ -49,6 +49,9 @@ def __init__(self, squash_handler, first_state_hash):
         self._last_in_batch = []
         self._last_state_hash = first_state_hash
 
+    def __del__(self):
```

Comment:
I don't see the __del__ method being called on the SerialScheduler (In CandidateBlock scheduler.cancel() is called, though). In general the only methods that should be called by external components are the ones defined in the scheduler.Scheduler abstract base class.